### DISCUSSION TITLE
Compiling the latest versions

### DISCUSSION THREAD

**Discussion by udo-munk** [udo-munk/z80pack#278] [Q&A]
Sometimes I get asked questions why something doesn't work, and this might be caused by using an older version and not the latest current. Sometimes people miss it, that the repository most of the time has two branches:

master - finished and final releases
dev      - latest sources still under development, but ususally stable

To use the latest dev version you need to do this:

git clone https://github.com/udo-munk/z80pack.git
cd z80pack
git checkout dev

You now will build everything from the latest dev branch and not some older finished release.

Hope this helps,
Udo

**Comment by ravn** [comment]
Missed this when writing the README.md.  You may want to add this.

**Reply by ravn** [reply]
Also the "fork repository" mechanism is geared towards working in "master" - nothing wrong with that , but you need to be aware of work actually happening in another branch so this is important knowledge

**Comment by udo-munk** [comment]
Thanks a lot and added the information.
